sdk/client-core/core/src/main/java/io/clientcore/core/implementation/util/Slf4jLoggerShim.java

```diff
@@ -204,16 +203,19 @@ private static Object createLogger(String className) {
             Object logger = LOGGER_FACTORY_GET_LOGGER_METHOD_HANDLE.invoke(className);
 
             if (NOP_LOGGER_CLASS.isAssignableFrom(logger.getClass())) {
-                DEFAULT_LOGGER.log(VERBOSE, "Resolved NOPLogger, SLF4J logging will be disabled.", null);
-
+                logErrorWithSlf4j(VERBOSE, "Resolved NOPLogger", null);
                 return null;
             }
 
             return logger;
         } catch (Throwable e) {
-            DEFAULT_LOGGER.log(WARNING, "Failed to create SLF4J logger, SLF4J logging will be disabled.", e);
-
+            logErrorWithSlf4j(WARNING, "Failed to create SLF4J logger", e);
             return null;
         }
     }
+
+    private static void logErrorWithSlf4j(ClientLogger.LogLevel level, String message, Throwable throwable) {
```

Comment:
Why is this method called `logErrorWithSlf4j()` if it uses the default logger?

Reply:
it reports an error that happened with slf4j, happy to rename if you have a better suggestion

Reply:
renamed